wrappers/python/simtk/openmm/app/internal/pdbstructure.py

```diff
@@ -123,7 +123,7 @@ class PdbStructure(object):
     """
 
 
-    def __init__(self, input_stream, load_all_models=False):
+    def __init__(self, input_stream, load_all_models=False, extraParticleIdentifier='EP'):
```

Comment:
Maybe `extraParticleElementName`, to make it clearer that it will appear in the `element` field of the ATOM record?